lib/action_cable/server/base.rb

```diff
@@ -36,11 +36,11 @@ def worker_pool
         @worker_pool ||= ActionCable::Server::Worker.pool(size: config.worker_pool_size)
       end
 
-      # Requires and returns an array of all the channel class constants in this application.
-      def channel_classes
+      # Requires and returns an hash of all the channel class constants in this application.
+      def channel_classes_hash
```

Comment:
I have updated the docs to say `channel class constants keyed by name`, I think that saying `keyed by channel name` could be confused with the `channel_name` method from #34